src/database/models/VerifiedUserModel.ts

```diff
@@ -1,9 +1,10 @@
 import {model, Schema} from 'mongoose';
 
 export const VerifiedUser = new Schema({
+  userId: Number,
   guildId: String,
```

Comment:
Can't users be verified in multiple guilds?

Reply:
In the previous DB setup, there would be a different collection for each guild's verified users. So if a user would be verified in multiple guilds there would be a different document for each guild they are verified in. To get similar functionality while keeping all verified users across all guilds in one collection, we would need to track a guildId along with the other information, and if the user is verified in multiple guilds, they will have multiple documents in this collection. EFfectively, its the same number of documents as before, but they are all in one collection.